tests/unit/core/selector.js

```diff
@@ -1,6 +1,10 @@
 define( [
 	"jquery",
-	"ui/core"
+	"ui/core",
```

Comment:
This looks wrong. We shouldn't depend on `ui/core` anywhere anymore. This is the only one.